pkg/scheduler/core/extender.go

```diff
@@ -254,7 +254,7 @@ func (h *HTTPExtender) send(action string, args interface{}, result interface{})
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
-		return fmt.Errorf("Failed %v with extender at URL %v, code %v", action, h.extenderURL, resp.StatusCode)
+		return fmt.Errorf("Failed %v with extender at URL %v, code %v", action, url, resp.StatusCode)
```

Comment:
The `url` is actually: `url := strings.TrimRight(h.extenderURL, "/") + "/" + action`, which included action. 

So it seems like the original version is fine. WDYT?

Reply:
I think the original version won't have the action as we are printing just `h.extenderURL` and  we would get something like `http://127.0.0.1:12346/scheduler`  always which would be confusing. Since we have different verbs and returning `http://127.0.0.1:12346/scheduler/bind` or `http://127.0.0.1:12346/scheduler/filter` would be better is what I believe.

Another thing to note is we don't have logging in this file :(.

Reply:
`Failed %v with extender ...` is printing the action. So I think the original version is fine to me :)

Reply:
The problem to me is extender URL. We have to look at the code to identify that is my concern. If someone is looking at the error message in the log, it is not clear which endpoint we are hitting. This is useful for debugging is what I think.